src/main/java/gui/RootPane.java

```diff
@@ -45,7 +45,7 @@ public void start(Stage primaryStage) throws Exception {
         // created. Also has some useful settings.
         //TODO: change this line of code.
         Scene scene = new Scene(topLevelPane);
-        scene.getStylesheets().add("testcss.css");
+        scene.getStylesheets().add("stylesheet.css");
         primaryStage.setScene(scene);
         primaryStage.setTitle("Hoi ben een titel lol.");
```

Comment:
These line note apply to the lines they are pinned to. I meant the title "hoi ben een titel lol." And I think it would be nice if that would be a string constant in the top of this class